examples/arkode/F2003_serial/ark_diurnal_kry_bp_f2003.f90

```diff
@@ -0,0 +1,541 @@
+! ------------------------------------------------------------------
+! Programmer(s): David J. Gardner, Cody J. Balos @ LLNL
+!                modified by Jean M. Sexton @ LBL
+!                modified by Daniel M. Margolis @ SMU
```

Comment:
You copy-pasted the author list [incorrectly] from some other file.  Please go back and double-check the original author list of *ALL* the F77 and F90 codes that you ported, and ensure that these F2003 versions properly reflect authorship.